### ISSUE TITLE
jslint options seem to have boolean logic errors

### ISSUE THREAD

**Issue by scottdensmore** [rondevera/jslintmate#6]
I wrote this code to test it out:

```
var j = 0;
var i = 1;
var _foo = 'Scott';

document.createElement("div");

for (i = 1; i < 100; i++) {
    j += i;
}
```

I was expecting complaints about the _ and ++ but that is fine. So I added this to the top of the file:

`/*jslint nomen: false, maxerr: 50, indent: 4 */`

And still did not get the error about dangling _.  I update it to true and it did correct itself. If you paste that code into jslint.com it works as expect... is this a bug in your implementation?

**Comment by scottdensmore** [comment]
Sorry.. I think the true / false is backwards for jslint.  Any thoughts?

**Comment by rondevera** [comment]
Hey Scott,

At some point in JSLint's recent history, Crockford flipped the behaviors of several options, which likely includes `nomen`. For example, `white: false` used to allow "sloppy" whitespace, but in the latest version of JSLint, you'd instead use `white: true`.

http://jslint.com/ uses the latest version of JSLint (as it should), while JSLintMate currently uses an older version of JSLint -- before these options were flipped around.

I have a local commit that updates to the latest stable JSLint build. The upside is that it makes JSLintMate consistent with http://jslint.com/ ; the downside, of course, is the sudden behavior changes! I'm not sure there's a smooth way to make this transition, but it'll be part of the upcoming v1.2 release.

If you'd like a quick local fix, you can grab [the latest stable JSLint build](http://jslint.com/jslint.js) manually, store it in your project (or home directory, etc.), then set the `--linter-file` option in your bundle; there's an example in the [JSLintMate readme](https://github.com/rondevera/jslintmate/#readme).

**Comment by scottdensmore** [comment]
when do you think 1.2 will be out?

**Comment by rondevera** [comment]
Sorry for the delay here. I've pushed my progress to the `development` branch; among other changes, this updates the onboard copies of JSLint and JSHint.

I have a few more features and fixes I'd like to land before tagging v1.2. This'll probably take another 2-4 weeks, but if you're interested in helping get there sooner, send me a message! (Or, of course, feel free to use the edge `development` build.)

**Comment by rondevera** [comment]
Hey Scott,

Thanks for your patience! JSLintMate 1.2 is now available, and has the latest stable builds of JSLint and JSHint. If you ever want to swap out the built-in copies of JSLint or JSHint for your own edge builds (or custom builds), check the ["JSLint/JSHint builds" section of the readme](https://github.com/rondevera/jslintmate/#readme). Hope this covers it!

http://rondevera.github.com/jslintmate/

Cheers,
Ron
